HHS/Head-Start-TTADP
# [PROD Fix FEI root cause for region 6, fix bug with save draft and uploading files, make goals distinct by root cause on the RTR, RTR profile tab, suspending objective modal

## Description of change

**TTAHUB-2119**: Fix for region 6 'Other ECE Care Options' root cause.
**TTAHUB-2020**: Add the "recipient leadership" table to the Profile tab of the Recipient record page
**TTAHUB-2085**: do not de-duplicate RTR goals based on responses
**TTAHUB-1925**: Suspend objectives from anywhere (and collect reasons)
**TTAHUB-2017**: fix form data loading
**TTAHUB-1567**: remove national center ARs

## How to test

View the profile tab on the RTR. Confirm the recipient personnel is present.
View the recipient /recipient-tta-records/706/region/1/goals-objectives on staging. Confirm they have 2! FEI goals
Suspend an objective from the RTR or the report. Confirm that the modal collects the reason and the reason displays under the report.

**2017**
To see bug:

- Create an AR
- Select an existing goal and objective, add a file
- Save draft
- Without clicking "Save goal" or anything else on the AR, click the "Activity reports" link on the side nav.
- Return to your report, note the file is missing. This PR fixes that bug.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2119
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2017
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1925
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2020
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2085
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1567

## Checklists

### Every PR



### Production Deploy

- [x] Pull pre-deploy db dump

### After merge/deploy
